proxy/testdata/testservice.proto

```diff
@@ -23,6 +23,7 @@ option go_package = "github.com/Snowflake-Labs/sansshell/proxy/testdata";
 message MyNested {
```

Comment:
might be also nice to have a 'oneof' where one of the options is sensitive. You're looping over the message fields with 'Range', so the implementation is correct, but it would be nice to have a test that exercises this behavior

Reply:
Added, and I've added a couple of other fields to test code paths that weren't yet being exercised by the tests.